go/test/endtoend/vreplication/resharding_workflows_v2_test.go

```diff
@@ -246,6 +246,28 @@ func TestBasicV2Workflows(t *testing.T) {
 	log.Flush()
 }
 
+func waitForWorkflowToStart(t *testing.T, ksWorkflow string) {
+	done := false
+	ticker := time.NewTicker(10 * time.Millisecond)
+	log.Infof("Waiting for workflow %s to start", ksWorkflow)
+	for {
+		select {
+		case <-ticker.C:
+			if done {
+				return
+			}
+			output, err := vc.VtctlClient.ExecuteCommandWithOutput("Workflow", ksWorkflow, "show")
+			require.NoError(t, err)
+			if strings.Contains(output, "\"State\": \"Running\"") {
```

Comment:
Minor nit, maybe better to use a RegEx or JSON parsing so that we are checking for what we really care about while resilient to whitespace and other minor format changes? Full JSON parsing is probably overkill, maybe RegEx is too :-) 

Feel free to ignore.